### ISSUE TITLE
Dont set focus on menu in select on open in react

### ISSUE THREAD

**Issue by FlyersPh9** [iTwin/iTwinUI#967]
If someone clicks on keyboard arrow down while the focus is on the trigger (dropdown button), the focus should go into the menu.  The option to set focus on the menu or not should be available.

**Comment by mayank99** [comment]
Just for clarity:
`setFocus` was added to partly solve this problem.
https://github.com/iTwin/iTwinUI/blob/b20fdecca7c5699169079f90b52bc5f52ddbc8db/packages/itwinui-react/src/core/Menu/Menu.tsx#L28-L32

The other half remains unsolved (moving focus to menu when pressing <kbd>↓</kbd>

**Comment by mayank99** [comment]
I think this feature request is not relevant anymore. The `Menu` component was removed, and `Select` should always move focus. Plus there is `Popover` which also moves focus.

The "move focus on arrow-down" idea is interesting, but hard to justify without a concrete use case. I will close this issue for now, but it can be reopened if we find a good use for this functionality.
